content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+
+![Figure 3: source dmtf.org/education](/img/fig3-resourcemap.png "Figure 3: source dmtf.org/education")
+
+### OEM extensions
+
+Computer makers have the possibility of modeling proprietary properties (not standardized) and constituting added value compared to the competition. For example, HPE iLO can store firmware updates before deploying them to their respective components. This specificity is described in the proprietary schema `#HpeiLOUpdateServiceExt` present in the `Oem.Hpe{}` object under the URI `/redfish/v1/UpdateService` (Figure 4).
+
+![Figure 4: OEM extension](/img/fig4-oemextensions.png "Figure 4: OEM extension")
+
+OEM extensions can also be used by a manufacturer while awaiting the standardization of a functionality unanimously recognized by DMTF members.
+
+## Resource collections
+
+As shown in figure 3 above, links contained in the Redfish root tree, do not provide direct access to the properties of the server subsystems. Instead, they point to “resource collections”. The collection members are described by a unique schema. For example, the response to the GET `/redfish/v1/Chassis` request made against the BMC of an HPE Superdome consisting of 2 chassis in a rack, is a collection containing four members: RackGroup, Rack1, r001u03b and r001u08b (Figure 5). These four members have the `#Chassis` diagram in common.
+
+![Figure 5: HPE Superdome chassis collection](/img/fig5-sdfchassis.png "Figure 5: HPE Superdome chassis collection")
+
+Another example: An HPE ProLiant server containing a storage enclosure connected to a (“modern”) storage controller is represented by two members in the chassis collection. A member pointing to the chassis containing the motherboard of the server (`/redfish/v1/Chassis/1`) and a member pointing to the backplane of the storage enclosure (`/redfish/v1/Chassis/DE040000`).
+
+Each member of the collection is an endpoint for accessing the member properties. Links to related resources constitute transversal access to other subsystems. For example, the storage chassis in Figure 6 contains a cross-link to the storage device under the "Systems" tree (`/redfish/v1/Systems/1/Storage/DE040000`).
+
+![Figure 6: Universal backplane model](/img/fig6-storagechassis.png "Figure 6: Universal backplane model")
+
+### Naming convention of collection members
+
+The naming of collection members is specific to each implementation of the Redfish® service. The BMC endpoint for an HPE server of type ProLiant is: `/redfish/v1/Managers/1`. That of an OpenBMC is: `/redfish/v1/bmc` and that of a Superdome is `/redfish/v1/Managers/RMC`. Dell, SuperMicro, Lenovo probably have their own way of naming their BMCs, while remaining compliant with the specifications.
```

Comment:
Modify the second sentence to read:
"The BMC endpoint for an HPE ProLiant server is:..."